### PR TITLE
FIX JRE centos8

### PR DESCRIPTION
**Pull request by AlvaroVega**
comes from https://github.com/telefonicaid/fiware-cygnus/pull/2114 and https://github.com/telefonicaid/fiware-cygnus/pull/2109

### CODE REVIEW

File: docker/cygnus-ngsi/Dockerfile [hunk 1]
```diff
@@ -327,9 +327,11 @@ RUN \
     cd ${CYGNUS_HOME}/cygnus-ngsi && \
     ${MVN_HOME}/bin/mvn ${MAVEN_ARGS} clean && \
     rm -rf /root/.m2 && rm -rf ${MVN_HOME} && rm -rf ${FLUME_HOME}/docs && rm -rf ${CYGNUS_HOME}/doc && rm -f /*.tar.gz && \
-    echo "INFO: Java runtime not needs JAVA_HOME... Unsetting..." && \
-    unset JAVA_HOME && \
-    yum erase -y git java-${JAVA_VERSION}-openjdk-devel python2 && \
+    # FIXME : Fix java errors and warnings
```
Comment (fgalan) [inline docker/cygnus-ngsi/Dockerfile:330]: I'd suggest to use issue #2133 for all fixes needed due to CentOS. In this sense:

```suggestion
        # FIXME #2113: Fix java errors and warnings. Original script was breaking in CentOS 8
```

Comment (mapedraza) [reply docker/cygnus-ngsi/Dockerfile:330]: I just created an Issue for this specific problem: #2116
```suggestion
    # FIXME #2116: Fix java errors and warnings. Original script was breaking in CentOS 8
```

**Comment by AlvaroVega** [comment]
In this point of the PR cygnus image builds and cygnus starts apparently without problems. IMHO we can merge it to reestablish latest image.

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**